app/viewmodels/SummaryCardHelper.scala

```diff
@@ -175,7 +175,7 @@ class SummaryCardHelper @Inject()(link: views.html.components.link) extends Impl
 
   private def lppCardBody(lpp: LatePaymentPenalty)(implicit messages: Messages): Seq[SummaryListRow] = {
     Seq(summaryListRow(
-      messages("summaryCard.key1"),
+      messages("summaryCard.lpp.key1"),
```

Comment:
Checked with designers, we want to reference 'VAT period' throughout the whole service. So we could change `summaryCard.key1` in the messages file to be:
`summaryCard.key1 = VAT period`

Reply:
implemented changes in [https://github.com/hmrc/penalties-frontend/pull/127](url)